pkg/api/handlers/get_cluster_credentials_oidc.go

```diff
@@ -33,7 +33,7 @@ func (d *getClusterCredentialsOIDC) Handle(params operations.GetClusterCredentia
 		return NewErrorResponse(&operations.GetClusterCredentialsDefault{}, 500, err.Error())
 	}
 
-	if !kluster.Spec.Dex {
+	if kluster.Spec.Dex == nil || !*kluster.Spec.Dex {
```

Comment:
using the `BoolValue(v *bool) bool` function from the swag package could help clean up all these if statements
```suggestion
	if ! swag.BoolValue(kluster.Spec.Dex) {
```

Reply:
Thanks. Nice tip!